pcsx2/Counters.h

```diff
@@ -85,17 +85,17 @@ struct SyncCounter
 //------------------------------------------------------------------
 // SPEED HACKS!!! (1 is normal) (They have inverse affects, only set 1 at a time)
 //------------------------------------------------------------------
-#define HBLANK_COUNTER_SPEED	1 //Set to '3' to double the speed of games like KHII
+#define HBLANK_COUNTER_SPEED	2 //Set to '3' to double the speed of games like KHII
 //#define HBLANK_TIMER_SLOWDOWN	1 //Set to '2' to increase the speed of games like God of War (FPS will be less, but game will be faster)
 
 //------------------------------------------------------------------
 // NTSC Timing Information!!! (some scanline info is guessed)
 //------------------------------------------------------------------
 #define FRAMERATE_NTSC			29.97 // frames per second
 
-#define SCANLINES_TOTAL_NTSC	525 // total number of scanlines
+#define SCANLINES_TOTAL_NTSC	263 // total number of scanlines
```

Comment:
Did you mean to account for the PSX mostly running in "240p" mode?
If so, it can and does use full NTSC resolution as well.

Reply:
Yeah pretty much, I was trying to account 240p so would the default pcsx2 value be correct since it can use full NTSC resolution?